File: UClient/Assets/Scripts/Lua/Common/Logger.lua
--[[
Author: fasthro
Date: 2020-11-25 12:21:31
Description: logger
--]]
local logger = {}

function logger.info(arg)
    UnityEngine.Debug.Log(tostring(arg))
end

function logger.debug(arg)
    UnityEngine.Debug.Log(tostring(arg))
end

function logger.warning(arg)
    UnityEngine.Debug.Log(tostring(arg))
end

function logger.error(arg)
    UnityEngine.Debug.Log(tostring(arg))
end

_G.logger = logger

File: LockstepServer/Lockstep/Scripts/Debug/Logger.cs
/*
 * @Author: fasthro
 * @Date: 2020/12/29 17:03:13
 * @Description:
 */

namespace Lockstep
{
    public static class Logger
    {
        public static ILogger logger;

        public static void Debug(object message)
        {
            logger?.Debug(message);
        }

        public static void Error(object message)
        {
            logger?.Error(message);
        }

        public static void Info(object message)
        {
            logger?.Info(message);
        }

        public static void Warning(object message)
        {
            logger?.Warning(message);
        }
    }
}